pkgs/by-name/co/configarr/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nodejs,
+  pnpm_10,
+  makeBinaryWrapper,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "configarr";
+  version = "1.13.4";
+
+  src = fetchFromGitHub {
+    owner = "raydak-labs";
+    repo = "configarr";
+    rev = "v1.13.4";
+    hash = "sha256-hfP1hrqK/ueuY4ll1Cr4msdJO0yu/mVTVYLP+u/xB5U=";
+  };
+
+  nativeBuildInputs = [
+    nodejs
+    pnpm_10.configHook
+    makeBinaryWrapper
+  ];
+
+  pnpmDeps = pnpm_10.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-NYq8nLML2C1ZgxecmchyeaC3sRsUSgRjwLto84yqboI=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    pnpm build
+
+    runHook postBuild
+  '';
+
+  checkPhase = ''
+    runHook preCheck
+
+    pnpm test
+
+    runHook postCheck
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+
+    install -D -t $out/share bundle.cjs
```

Comment:
The `-D` option of install implicitly includes creating the target directory (like `mkdir $out`)

Reply:
Yes, that's right. In fact, it's explicit (that's the express purpose of the `-D` option).